conf/application.conf

```diff
@@ -66,6 +66,10 @@ cache {
   expiryInMinutes = 40
 }
 
+tracking-consent-frontend {
+  gtm.container = "a"
+}
```

Comment:
We need to override this value in app-config-xxx. Do you know what values we need to provide for every environment?